### DISCUSSION TITLE
È possibile visionare le segnalazione scadute?

### DISCUSSION THREAD

**Discussion by MonicaMandelli** [globaleaks/globaleaks-whistleblowing-software#4243] [Q&A] [answered]
Volevo chiedere dove è possibile visionare le segnalazione scadute? E come fare per poterle visionare?

**Comment by evilaliv3** [comment, marked as answer]
Buonasera Monica,

La ringraziamo per la domanda.

Le segnalazioni scadute per ragioni di privacy sono cancellate e sovrascritte.
Per le stesse il sistema mantiene un solo registro degli eventi avvenuti a sistema visibile al momento ai soli amministratori.

Posso.chiederle di chiarirci meglio la sua necessità?

Grazie

**Comment by MonicaMandelli** [comment]
Per le seguenti ragioni:

  1.  La normative privacy mi impone di mantenere per 5 anni le registrazioni avvenute;
  2.  La necessità di ricostruire lo storico in qualità di Gestore delle Segnalazioni;
  3.  Questa settimana è erroneamente scaduta una segnalazione e a seguito di un problema alla rete internet e al PC non è stato possibile sistemarla

Monica

**Comment by evilaliv3** [comment]
La ringrazio.
Se con normativa privacy riferisce la normativa legge 24/2023, le faccio presente che la legge impone solo termine massimo di 5 anni, non un termine minimo.

Ad ogni modo purtroppo devo confermale che ciò che scade o che viene cancellato da un utente viene cancella to in modo indelebile
